sources/emscripten/0_niLang_shell.html

```diff
@@ -181,6 +181,7 @@
 
       console.log('... createDefaultNiModule');
       var Module = createDefaultNiModule({
+        UseInput: true,
```

Comment:
This is the default that makes sense for me.

Reply:
In the shell template yes.